src/client/render/render-core.js

```diff
@@ -61,6 +64,9 @@ let renderCore = function() {
 		activeCamera: null,
 		freeCamera: null,
 		trackingCamera: null,
+		shaderUniforms: {
+			"time": { value: 1.0 }
```

Comment:
I missed this last time, but since this value represents the time elapsed, it should probably start at 0. This really doesn't change anything but eh.